src/dev/pages/button-area/button-area.ejs

```diff
@@ -0,0 +1,20 @@
+<forge-card outlined="true">
```

Comment:
Super minor, but can we remove the `="true"` part of this attribute? I know it doesn't matter, but technically with boolean attribute just the _existence_ matters so I have been trying to remove instances of this to avoid setting the example.